components/service/telemetry/src/main/java/org/mozilla/telemetry/metrics/TelemetryMetrics.java

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.telemetry.metrics;
+
+import android.os.SystemClock;
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.RestrictTo;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.mozilla.telemetry.TelemetryHolder;
+import org.mozilla.telemetry.util.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * TelemetryEvent specifies a common events data format, which allows for broader, shared usage of
+ * data processing tools.
+ */
+public class TelemetryMetrics {
```

Comment:
This looks like a clone of the events class?

This document describes how the metrics will look like:
https://docs.google.com/document/d/1q2V45_fyzm_DGaCos34QEQimsodYJMJCq5Ix-ys9KkE/edit

GeckoView will add an API that will give us the data for "histograms", "keyedHistograms", "scalars" and "keyedScalars".